relm4-examples/libadwaita/examples/view-switcher.rs

```diff
@@ -0,0 +1,75 @@
+use adw::traits::ApplicationWindowExt;
+use gtk::prelude::{BoxExt, ButtonExt, GtkWindowExt, OrientableExt};
+use relm4::{send, AppUpdate, Model, RelmApp, Sender, WidgetPlus, Widgets};
+
+#[derive(Default)]
+struct AppModel {
+    counter: u8,
+}
+
+enum AppMsg {
+    Increment,
+    Decrement,
+}
+
+impl Model for AppModel {
+    type Msg = AppMsg;
+    type Widgets = AppWidgets;
+    type Components = ();
+}
+
+impl AppUpdate for AppModel {
+    fn update(&mut self, msg: AppMsg, _components: &(), _sender: Sender<AppMsg>) -> bool {
+        match msg {
+            AppMsg::Increment => {
+                self.counter = self.counter.wrapping_add(1);
+            }
+            AppMsg::Decrement => {
+                self.counter = self.counter.wrapping_sub(1);
+            }
+        }
+        true
+    }
+}
+
+fn application_window() -> adw::ApplicationWindow {
+    adw::ApplicationWindow::builder().build()
+}
+
+#[relm4_macros::widget]
+impl Widgets<AppModel, ()> for AppWidgets {
+    view! {
+        main_window = application_window() -> adw::ApplicationWindow {
+            set_default_width: 300,
+            set_default_height: 200,
+
+            set_content = Some(&gtk::Box) {
+                set_orientation: gtk::Orientation::Vertical,
+                set_margin_all: 5,
+                set_spacing: 5,
+
+                append = &adw::HeaderBar {
+                    set_title_widget = Some(&adw::ViewSwitcherTitle) {
+                        set_title: "A stack switcher App",
+                        set_stack: Some(&stack),
+                    }
+                },
+                
+                append: stack = &adw::ViewStack {
+                    add_titled: args!(
+                        &gtk::Label::new(Some("This is the start page")), Some("First"), "First Page"),
```

Comment:
Maybe you could help me on that:
I want to create those subpages, which use multiple arguments. Now I want to continue below with the macro notation. Do need to use components or such? Or is there something I am missing so far?

Reply:
@tronta you should be able to use `add_titled(arg1, arg2, ...) = &gtk::Label ... `

Reply:
The first parameter of the function is expecting the label, so I'm not sure how I can assign it as you proposed.

Reply:
Indeed, it's working like this. You have probably described it somewhere, but I was not expecting this behavior. Maybe I forgot this. Thanks